config/default.yaml

```diff
@@ -23,11 +23,9 @@ services:
   case:
     url: 'https://ccd-data-store-api-prl-ccd-definitions-pr-543.service.core-compute-preview.internal'
   cos:
-    url: 'https://prl-cos-pr-513.service.core-compute-preview.internal'
+    url: 'https://prl-cos-pr-540.service.core-compute-preview.internal'
   documentManagement:
     url: 'https://ccd-case-document-am-api-prl-ccd-definitions-pr-543.service.core-compute-preview.internal'
-  dgs:
```

Comment:
@SangeethaPonnusamy Why we have removed this config ?

Reply:
I haven't removed this specifically. May be while rebasing it might have happened. Added this back